src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/CacheParameter.cs

```diff
@@ -8,14 +8,14 @@
 
 namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects
 {
-    public class CacheParameter : ICacheParameter, IAllowDefaultIfOptionWithoutValue
+    internal class CacheParameter : ICacheParameter, IAllowDefaultIfOptionWithoutValue
     {
-        public CacheParameter(string dataType, string? defaultValue, string? displayName, string? description)
+        internal CacheParameter(string dataType, string? defaultValue, string? displayName, string? description)
```

Comment:
For this project, it is already done. But for others: do we have to mark members internal if the type is already internal?

Reply:
In general or as part of this work/PR? IMO... in general yes, because thats what CodeAnalzyers are suggesting... but for this PR, it feels like it would be spam without added value...

Reply:
No, it's not mandatory. I just used replace to do it. I can revert it, i prefer to use internal modifiers where possible if class is internal.